apps/teamsjs-diagnostic-app/infra/azure.bicep

```diff
@@ -0,0 +1,132 @@
+param resourceBaseName string
```

Comment:
@ava-vellozzi I'm not familiar with what this file is for, can you tell me a little about what it's doing?

Reply:
To be honest, I'm not 100% sure - my app is built off of the Teams Toolkit "Tab App - React with FluentUI" so that was an autogenerated file. From what I understand, it involves setting up Azure resources and also handles authentication through AAD.
[https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview?tabs=bicep](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview?tabs=bicep)